### PR TITLE
new event at stylelabs

### PR DESCRIPTION
**Pull request by KrisvanderMast**

### CODE REVIEW

**Comment by maartenba** [comment]
@KrisvanderMast @tomkerkhove Kan je deze rebasen op huidige master branch? Sleept nogal wat bagage mee zo te zien :-)

File: _posts/events/2017-10-19---Azure-from-the-trenches.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout: single
+title: "2017-10-19 - Azure from the trenches"
+date: 2017-06-01 13:42:00 +0000
```
Comment (tomkerkhove) [inline _posts/events/2017-10-19---Azure-from-the-trenches.md]: Don't forget to update this one

**Comment by maartenba** [comment]
Still a lot of old stuff in this PR. Try a fresh fork and then replay the last chance that adds the event?

**Comment by maartenba** [comment]
Manually cherry-picked the changes in. Thanks, @kvdm!

**Pull request closed without merging**